packages/models-library/src/models_library/services_base.py

```diff
@@ -20,7 +20,7 @@ class Config:
         frozen = True
 
 
-class ServiceBase(BaseModel):
+class ServiceDisplay(BaseModel):
```

Comment:
ServiceBaseInfo? Display in this context of a backend service is a bit out of context no?